ios/brave-ios/Sources/AIChat/Components/AIChatLoaderView.swift

```diff
@@ -75,6 +75,71 @@ struct DotsProgressViewStyle: ProgressViewStyle {
   }
 }
 
+struct CircularProgressViewStyle: ProgressViewStyle {
+  private let thickness: Double
+  private let foregroundColor: Color
+  private let backgroundColor: Color
```

Comment:
Delete and use `foregroundStyle`/`backgroundStyle` modifiers at call site instead then you can just do `Circle().stroke(.foreground)` or `Circle().stroke(.background)`.  You could also swap out foregroundStyle for `tint` modifiers and use `.stroke(.tint)` as well but its mostly semantics at that point